.github/workflows/tests.yml

```diff
@@ -73,55 +76,57 @@ jobs:
           || { echo "***** TESTS FAILED. Attempting retry."; $RUN_TESTS; } \
           || { echo "***** TESTS FAILED. Attempting final retry."; $RUN_TESTS; }
 
-  # This step can be uncommented to support feature development.  When developing a 
-  # feature which spans multiple repos, you can update the poms to use the latest 
-  # SNAPSHOT across the board, and run this action as a sanity check to ensure 
+  # This step can be uncommented to support feature development.  When developing a
+  # feature which spans multiple repos, you can update the poms to use the latest
+  # SNAPSHOT across the board, and run this action as a sanity check to ensure
   # everything is working in unison.
 
-  # # Build the code and run the microservice unit/integration tests.
-  # build-and-test-microservices:
-  #   runs-on: ubuntu-latest
-  #   steps:
-  #     - name: Checkout Code
-  #       uses: actions/checkout@v3
-  #       with:
-  #         submodules: 'recursive'
-  #     - name: Set up JDK ${{env.JAVA_VERSION}}
-  #       uses: actions/setup-java@v3
-  #       with:
-  #         distribution: ${{env.JAVA_DISTRIBUTION}}
-  #         java-version: ${{env.JAVA_VERSION}}
-  #     - uses: actions/cache@v3
-  #       with:
-  #         path: ~/.m2/repository
-  #         key: ${{ runner.os }}-maven-build-${{ hashFiles('**/pom.xml') }}
-  #         restore-keys: |
-  #           ${{ runner.os }}-maven-build-
-  #           ${{ runner.os }}-maven-format-
-  #           ${{ runner.os }}-maven-
-  #     - name: Build Project
-  #       env:
-  #         USER_NAME: ${{ secrets.USER_NAME }}
-  #         ACCESS_TOKEN: ${{ secrets.ACCESS_TOKEN }}
-  #       run: |
-  #         BUILD="mvn -s $GITHUB_WORKSPACE/.github/workflows/settings.xml -V -B -e -Pdev,examples,assemble,spotbugs -Dmaven.build.cache.enabled=false -DskipServices -Ddeploy -Ddist -T1C -pl "-:config-service" clean install -DskipTests"
-  #         $BUILD
-  #     - name: Run Microservice Unit Tests
-  #       env:
-  #         USER_NAME: ${{ secrets.USER_NAME }}
-  #         ACCESS_TOKEN: ${{ secrets.ACCESS_TOKEN }}
-  #       run: |
-  #         RUN_TESTS="mvn -s $GITHUB_WORKSPACE/.github/workflows/settings.xml -V -B -e -Dmaven.build.cache.enabled=false verify"
-  #         cd microservices
-  #         $RUN_TESTS \
-  #           || { echo "***** TESTS FAILED. Attempting retry."; $RUN_TESTS; } \
-  #           || { echo "***** TESTS FAILED. Attempting final retry."; $RUN_TESTS; }
+  # Build the code and run the microservice unit/integration tests.
+  build-and-test-microservices:
```

Comment:
Is the intention to leave this uncommented moving forward?

Reply:
This is going to have to go away actually.  I had something similar for the spring-boot-2.x branch, and it's a nice idea, but unless everything is a snapshot version it doesn't really add much value.  I would prefer to comment it out AFTER this is merged just to make sure we are running this github action right up until the last possible second.